types/compress.js/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for compress.js 1.1
+// Project: - (link to the repo is missing)
+// Definitions by: Yuri Drabik <https://github.com/yurist38>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class Compress {
```

Comment:
It seems like the [underlying library](https://www.npmjs.com/package/compress.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

Reply:
@PranavSenthilnathan Thanks for the remark, I've fixed it. Hope it's fine now, please take a look. Thanks!